tensorflow_addons/utils/test_utils.py

```diff
@@ -21,6 +21,7 @@
 import pytest
 import tensorflow as tf
 
+from tensorflow_addons import options
```

Comment:
if you use https://github.com/tensorflow/addons/blob/master/tensorflow_addons/options.py#L36-L40 seems it works.